### ISSUE TITLE
Just for the confirmation, does toolkit apply denoising when fine tuning?

### ISSUE THREAD

**Issue by mani-rai** [pytorch/fairseq#2966]
This question is just for confirmation. Like BERT, denoising doesn't apply when fine tuning, right? 

I was little bit confused because mbart guide gives following command to fine tune.

> fairseq-train path_2_data \
  --encoder-normalize-before --decoder-normalize-before \
  --arch mbart_large --layernorm-embedding \
  --task translation_from_pretrained_bart \
  --source-lang en_XX --target-lang ro_RO \
  --criterion label_smoothed_cross_entropy --label-smoothing 0.2 \
  --optimizer adam --adam-eps 1e-06 --adam-betas '(0.9, 0.98)' \
  --lr-scheduler polynomial_decay --lr 3e-05 --min-lr -1 --warmup-updates 2500 --total-num-update 40000 \
  --dropout 0.3 --attention-dropout 0.1 --weight-decay 0.0 \
  --max-tokens 1024 --update-freq 2 \
  --save-interval 1 --save-interval-updates 5000 --keep-interval-updates 10 --no-epoch-checkpoints \
  --seed 222 --log-format simple --log-interval 2 \
  --restore-file $PRETRAIN \
  --reset-optimizer --reset-meters --reset-dataloader --reset-lr-scheduler \
  --langs $langs \
  --ddp-backend no_c10d

Here we can see there's no fine tuning argument given whereas in [fairseq doc](https://fairseq.readthedocs.io/en/latest/command_line_tools.html#fairseq-train)  there's fine tuning argument we can apply.

**Comment by mani-rai** [comment]
Confirmed denosing doesn't applies while fine tuning. Found this confirmation in BART document.
